oca_dependencies.txt

```diff
@@ -0,0 +1,2 @@
+# See https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#oca_dependencies-txt
+partner-contact https://github.com/OCA/partner-contact 15.0
```

Comment:
This is not used anymore AFAIK

Reply:
@ivantodorovich should I remove the whole file?

Reply:
YEEEEEEEEEEEEEEEEEEEES :tada: :partying_face: